js/source.js

```diff
@@ -20,3 +20,22 @@ var expEntries = [
     writeup:  'Marketing Coordinator developing proposals and marketing material for an engingeering and environmental sciences firm.'
   }
 ];
+
+
```

Comment:
I do like that you combined both object arrays onto the source.js file. Makes it easy to go and find that information in one place for easy editing.